File: 단계별로_풀어보기/7. 2차원 배열/1_2738.js
// 입력
const fs = require('fs')
const inputArr = fs.readFileSync(0, 'utf-8').toString().trim().split('\n')

// 입력 파싱
const [N, M] = inputArr[0].split(' ').map(Number)

const A = inputArr.slice(1, N + 1).map(row => row.split(' ').map(Number))
const B = inputArr.slice(N + 1).map(row => row.split(' ').map(Number))

// 배열 열산
const resultArr = A.map((row, rowIdx) => {
  return row.map((val, colIdx) => val + B[rowIdx][colIdx])
})

// 출력
let result = ''
resultArr.forEach((row, rowIdx) => {
  let ret = row.join(' ')

  if (rowIdx < N - 1) {
    ret += '\n'
  }

  result += ret
})

process.stdout.write(result)

File: 단계별로_풀어보기/7. 2차원 배열/1_2738.py
import sys

input = sys.stdin.readline
output = sys.stdout.write

N, M = map(int, input().split())

A = list(list(map(int, input().split())) for i in range(N))
B = list(list(map(int, input().split())) for i in range(N))

resultArr = [[A[i][j] + B[i][j] for j in range(M)] for i in range(N)]

for row in resultArr:
  output(f"{' '.join(str(elem) for elem in row)}\n")
